specification/eventgrid/data-plane/Microsoft.Web/stable/2018-01-01/Web.json

```diff
@@ -0,0 +1,679 @@
+              "type": "string"
+            }
+          }
+        },
+        "name": {
+          "description": "name of the web site that had this event.",
+          "type": "string"
+        },
+        "clientRequestId": {
+          "description": "The client request id generated by the app service for the site API operation that triggered this event.",
+          "type": "string"
+        },
+        "correlationRequestId": {
+          "description": "The correlation request id generated by the app service for the site API operation that triggered this event.",
+          "type": "string"
+        },
+        "requestId": {
+          "description": "The request id generated by the app service for the site API operation that triggered this event.",
+          "type": "string"
+        },
+        "address": {
+          "description": "HTTP request URL of this operation.",
+          "type": "string"
+        },
+        "verb": {
+          "description": "HTTP verb of this operation.",
+          "type": "string"
+        }
+      }
+    },
+    "WebSlotSwapWithPreviewCancelledEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for an Microsoft.Web.SlotSwapWithPreviewCancelled event.",
+      "type": "object",
+      "properties": {
+        "appEventTypeDetail": {
+          "description": "Detail of action on the app",
+          "type": "object",
+          "properties": {
+            "action": {
+              "description": "Type of action of the operation.",
+              "type": "string"
+            }
+          }
+        },
+        "name": {
+          "description": "name of the web site that had this event.",
+          "type": "string"
+        },
+        "clientRequestId": {
+          "description": "The client request id generated by the app service for the site API operation that triggered this event.",
+          "type": "string"
+        },
+        "correlationRequestId": {
+          "description": "The correlation request id generated by the app service for the site API operation that triggered this event.",
+          "type": "string"
+        },
+        "requestId": {
+          "description": "The request id generated by the app service for the site API operation that triggered this event.",
+          "type": "string"
+        },
+        "address": {
+          "description": "HTTP request URL of this operation.",
+          "type": "string"
+        },
+        "verb": {
+          "description": "HTTP verb of this operation.",
+          "type": "string"
+        }
+      }
+    },
+    "WebAppServicePlanDeletedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for an Microsoft.Web.AppServicePlanDeleted event.",
+      "type": "object",
+      "properties": {
+        "appServicePlanEventTypeDetail": {
+          "description": "Detail of action on the app service plan",
+          "type": "object",
+          "properties": {
```

Comment:
Another type of anonymous class where the content pattern is regular later in this file